pkg/types/alibabacloud/machinepool.go

```diff
@@ -76,3 +76,30 @@ func (a *MachinePool) Set(required *MachinePool) {
 
 // DefaultDiskCategory holds the default Alibaba Cloud disk type used by the ECS.
 const DefaultDiskCategory DiskCategory = CloudESSDDiskCategory
+
+// DefaultDiskSize holds the default Alibaba Cloud system disk size used by the ECS.
```

Comment:
```suggestion
/ DefaultSystemDiskSize holds the default Alibaba Cloud system disk size used by the ECS.
```

Reply:
Thank you for your suggestion, I have updated